helm/charts/kratos/templates/deployment-kratos.yaml

```diff
@@ -133,7 +133,7 @@ spec:
             {{- range $method, $methodEntry := .Values.kratos.emailTemplates }}
             {{- range $result, $resultEntry := $methodEntry }}
             - name: {{ include "kratos.name" $root }}-template-{{ $method }}-{{ $result }}-volume
-              mountPath: /conf/courier-templates/{{ $method }}/{{ $result }}
+              mountPath: /conf/courier-templates/{{ $method | snakecase }}/{{ $result }}
```

Comment:
```suggestion
              mountPath: /conf/courier-templates/{{ $method | kebabcase }}/{{ $result }}
```

Reply:
Hmm, the `mountPath` has to be snake_case though because Kratos expects template name to be snake_case. This is what caused me to make this PR, otherwise it will work without fix :)